docs/articles/install/3-install-with-helm.md

```diff
@@ -139,10 +142,29 @@ Testkube requires a variety of secrets to operate. Any required secret that is n
 - testkube-minio-credentials
 - testkube-license
 
+### Testkube License
+
+Enter your license key which can be either an offline or online license. Replace «license» with your license.
+
+For an online license, the format is `XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-V3`.
+
+```
+kubectl create secret generic testkube-license --from-literal=LICENSE_KEY=«license»
+```
+
+For an offline license, the format is a long string prefixed with `key/`.
+
+```
+kubectl create secret generic testkube-license --from-literal=license.lic=«license»
+```
+
+This secret is referenced by the `global.enterpriseLicenseRef` setting. For offline licences, you will also have to set `global.enterpriseOfflineAccess: true`.
```

Comment:
The Madrid/Poznan staging environment are using offline licenses and do not set this variable, I need to check if this is necessary really: https://github.com/kubeshop/testkube-deployment/blob/main/bundles/testkube-madrid-stage/templates/testkube-control-plane.yaml#L19

🤔